src/logic/calculate.js

```diff
@@ -40,12 +39,13 @@ const calculate = (calculatorObj, buttonName) => {
             calculatorObj.operation = null;
             break;
         default:
-            if(next){
+            if(next !== '0' && next){
                 calculatorObj.next += buttonName;
             } else {
                 calculatorObj.next = buttonName;
             }
     }
+    console.log(calculatorObj)
```

Comment:
please remove log messages, the one in operate too.

Reply:
Done